OpenRA.Game/Activities/Activity.cs

```diff
@@ -266,6 +282,11 @@ public virtual IEnumerable<Target> GetTargets(Actor self)
 		{
 			yield break;
 		}
+
+		public virtual TargetLineNode TargetLineNode(Actor self)
```

Comment:
Can you give me an example? All compositie activities gets "unrolled" and I had no trouble cycling through all of them with Next.